Frameworks/AddressBook/ABContactInternal.mm

```diff
@@ -28,4 +28,18 @@ - (id)initWithContact:(WACContact*)contact {
     return self;
 }
 
++ (NSDate*)convertDate:(WACContactDate*)date {
```

Comment:
Since it is a category, it will require either a category force-include or manual inclusion in the .dll and .UnitTests.exe.

---
In reply to: [75180351](https://github.com/Microsoft/WinObjC/pull/884#discussion_r75180351) [](ancestors = 75180351,75170063,75166689,75164209,75146840)